src/VisualStudio/IntegrationTest/TestUtilities/Common/ProjectUtilities.cs

```diff
@@ -9,10 +9,13 @@ public abstract class Identity
 
     public class Project : Identity
     {
-        public Project(string name)
+        public Project(string name, string relativePath = null)
         {
             Name = name;
+            RelativePath = relativePath;
         }
+
+        public string RelativePath { get; }
```

Comment:
@tmeschter Most of the time, the relative path will be <ProjectFolder>\<ProjectName>.<csproj\vbproj> . The relative path will be just Name.csproj if we are opening just the project which is not attached to a solution